init-wifi.h

```diff
@@ -1,6 +1,8 @@
 #ifndef __INIT_WIFI__
 #define __INIT_WIFI__
 
+#include <string.h>
```

Comment:
where do we need this?

Reply:
This header includes declaration of strcpy, memset and etc.

Reply:
but we never needed it - so why do we need it here?
isnt that included smwhere in Arduino.h?

Reply:
Yes, this is included in Arduino.h
I`ll remove this include from current file.